packages/dashboard/src/pages/Settings/components/DataSourceSetting.tsx

```diff
@@ -0,0 +1,20 @@
+import { MenuItem } from '@material-ui/core'
+import SettingSelect from './SettingSelect'
+import { useTrendingDataSource } from '../api'
+import { Services } from '../../../API'
+
+export default function DataSourceSetting() {
+    let source = useTrendingDataSource()
+    const handleChange = (event: any) => {
+        const value = event.target.value
+        Services.Settings.setTrendingDataSource(value)
+    }
+
+    return (
+        <SettingSelect value={source} onChange={handleChange}>
+            <MenuItem value={0}>CoinGecko</MenuItem>
+            <MenuItem value={1}>CoinMarketCap</MenuItem>
+            <MenuItem value={2}>Uniswap Info</MenuItem>
```

Comment:
The same as above, use the enum from `DataProvider`. (It is not defined in `shared` package. So you can leave a TODO here, once #3215 has merged, DataProvider enum should be moved in that package)